### ISSUE TITLE
Server requests authentication using unknown plugin sha256_password

### ISSUE THREAD

**Issue by Tino-F** [sidorares/node-mysql2#1082]
I'm trying to connect to my DB, but I get the below error message:

`Error: Server requests authentication using unknown plugin sha256_password. See TODO: add plugins doco here on how to configure or author authentication plugins.`

The code I'm using is as followed:
```js
const mysql = require('mysql2');
const pool = mysql.createPool({
  host: process.env.SQL_HOST,
  user: process.env.SQL_USER,
  port: process.env.SQL_PORT || 3306,
  password: process.env.SQL_PASS,
  database: 'MyDatabase',
  waitForConnections: true,
  connectionLimit: process.env.MAX_SQL_CONNECTIONS || 5,
  queueLimit: 0
});
const promisePool = pool.promise();
```

"mysql2": "^2.0.2",

**Comment by sidorares** [comment]
Hi @Tino-F , please read comments below https://github.com/sidorares/node-mysql2/pull/1021#issuecomment-559341452

tldr - `sha256_password` is currently not supported, it is recommended that you upgrade server to use `caching_sha2_password`

**Comment by sidorares** [comment]
( error message is not great though, any help improving is is welcomed )

**Comment by Tino-F** [comment]
@sidorares Thanks for the quick response

**Comment by kayjlou** [comment]
Make sure in your .env file you don't have quotations around your actual inputs (on the right side)

**Comment by InocentumAbhijit** [comment]
> Make sure in your .env file you don't have quotations around your actual inputs (on the right side)

I missed the `.env` file, solved 👍
